### ISSUE TITLE
Other YOAST plugins compatibility/features?

### ISSUE THREAD

**Issue by allanrehhoff** [senlin/so-clean-up-wp-seo#91]
Hey.

Not sure if it's within the scope of this plugin, so please let me know if it's a wontfix. :)
But otherwise I think it'd be great to implement some kind of compatibility layer if other certain YOAST plugins are active.

**The problem**
YOAST Recently consumed, something that used to be a decent plugin, 'Duplicate Post' and is now called 'Yoast Duplicate Post'
The YOAST Team adds new "features" faster than I change underwear, so inevetibly I'm now seeing bloatware notices in my backend (See screenshots)

**The solution**
This notice has the attribute `id="duplicate-post-notice"` so a `display: none; ` would be able to fix this, ideally with an additional option to choose what to hide from other yoast plugins.

**Steps to reproduce**
1. Install the newest version of 'Yoast Duplicate Post'
or
2. Install an old version, and update to newest through backend

Mine always appear after updating this plugin on any given client side.

**Attachments**
![2020-09-23_08-56-43](https://user-images.githubusercontent.com/24595705/93978332-1f3bb480-fd7c-11ea-93b9-ddceefcba666.png)

**Comment by senlin** [comment]
Hi @allanrehhoff 

I hear you and although I understand what you mean, this promotion is "only" to sign up for an email newsletter, not to sell anything like the Yoast SEO plugin does.

If I would add this to the Hide SEO Bloat plugin now, then that will open the floodgates to add anything and everything Yoast does, which in turn would mean that I would have to scour the internets on a daily basis to see all the crap Yoast adds to all plugins in their control. I simply have no interest in that.

Maybe you (or someone else) can develop a plugin that takes care of the rest of Yoast's Bloat?

Long story short: "wontfix", sorry.

**Comment by allanrehhoff** [comment]
Yeah I can definitely follow that logic.

Maybe someday someone will be up for undertaking such a daunting task.
